docs/diagrams/StudentClassDiagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+
+Class Student
+Class StudentName
+Class StudentPhone
```

Comment:
I think this should be changed to `Phone` instead, since the class to store a student's phone number is `Phone`? Same for `ParentPhone`, but I'm not too sure about this 😅  If so, then the multiplicities will have to be updated accordingly too